app/models/work.rb

```diff
@@ -0,0 +1,10 @@
+class Work < ApplicationRecord
+  has_many :votes, dependent: :destroy
+  validates :title, presence: true
+  validates :title, uniqueness: true
+
+  def self.by_votes
```

Comment:
You can use 

```ruby
Work.left_outer_joins(:votes).distinct.select('works.*, COUNT(votes.*) AS num_votes').group('works.id').order('num_votes DESC')
```

This would let Postgres do the sorting.